src/libraries/tests.proj contains incorrect condition

The test run complains:
`F:\git\runtime4\src\libraries\tests.proj(24,11): error MSB4092: An unexpected token "==" was found at character position 25 in condition "'$(Coverage)' == 'true' == 'true' and '$(SkipCoverageReport)' != 'true'".`

--- a/src/libraries/tests.proj
+++ b/src/libraries/tests.proj
@@ -21,7 +21,7 @@
   </ItemGroup>
 
   <Target Name="GenerateCoverageReport"
-          Condition="'$(Coverage)' == 'true' == 'true' and '$(SkipCoverageReport)' != 'true'"
+          Condition="'$(Coverage)' == 'true' and '$(SkipCoverageReport)' != 'true'"
           AfterTargets="Test"
           Inputs="$(CoverageReportInputPath)"
           Outputs="$(CoverageReportResultsPath)">

